docs/workbook.md

```diff
@@ -0,0 +1,61 @@
+  # Workbook
```

Comment:
```suggestion
# Workbook
```

Reply:
How did you spot that 😅 
![small](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExMjIzcjE0OHJ1ajc3ZXBnNm9xYmdvdjBnNnVpYnZlN2hnYTBxdGNuNSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/H8ar8GOTxqeJ3eLVgR/giphy.gif)